go.mod

```diff
@@ -2,4 +2,4 @@ module github.com/Luzifer/go-openssl/v4
 
 go 1.14
```

Comment:
You are using `io.ReadFull` below which in 1.14 was `ioutil.ReadFull`, please bump to `1.20` as minimum supported version.